docs/api.rst

```diff
@@ -3057,6 +3057,8 @@ Querying all supported assets
    :reqjson string name: The name of asset to be used to filter the result data. Optional.
    :reqjson string symbol: An asset symbol to be used to filter the result data. Optional.
    :reqjson string asset_type: The category of an asset to be used to filter the result data. Optional.
+   :reqjson string evm_chain: The name for the evm chain of the asset(s) to be used to filter the result data. Possible values are ``ethereum``, ``optimism``, etc. Optional.
+   :reqjson string address: The address of the evm asset to be used to filter the result data. Optional.
    :reqjson bool show_user_owned_assets_only: A flag to specify if only user owned assets should be returned. Defaults to ``"false"``. Optional.
```

Comment:
Can you add to the docs the constraint that if chain is set then asset type can only be evm token?